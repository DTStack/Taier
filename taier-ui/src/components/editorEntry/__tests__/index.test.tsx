import { render } from '@testing-library/react';
import EditorEntry from '..';

jest.mock('@dtinsight/molecule/esm/services/keybinding', () => ({
    KeybindingHelper: {
        queryGlobalKeybinding: jest.fn((id) => [id]),
        convertSimpleKeybindingToString: jest.fn((id) => `${id}`),
    },
}));

jest.mock('@dtinsight/dt-utils/lib', () => ({
    Utils: {
        isMacOs: jest.fn(() => false),
    },
}));

describe('Test EditorEntry', () => {
    it('Should match snapshot', () => {
        expect(render(<EditorEntry />).asFragment()).toMatchSnapshot();
    });
});
