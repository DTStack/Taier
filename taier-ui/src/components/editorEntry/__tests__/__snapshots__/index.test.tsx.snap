// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test EditorEntry Should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="entry"
  >
    <img
      class="logo"
      src="images/taier.png"
      width="200"
    />
    <div
      class="commands"
    >
      <div
        class="command"
      >
        <div
          class="label"
        >
          切换侧边栏
        </div>
        <div
          class="keybindings"
        >
          <code
            class="keyCode"
          >
            sidebar
          </code>
        </div>
      </div>
      <div
        class="command"
      >
        <div
          class="label"
        >
          切换面板
        </div>
        <div
          class="keybindings"
        >
          <code
            class="keyCode"
          >
            workbench.action.showPanel
          </code>
        </div>
      </div>
      <div
        class="command"
      >
        <div
          class="label"
        >
          运行 SQL
        </div>
        <div
          class="keybindings"
        >
          <code
            class="keyCode"
          >
            RunSQL
          </code>
        </div>
      </div>
      <div
        class="command"
      >
        <div
          class="label"
        >
          切换主题颜色
        </div>
        <div
          class="keybindings"
        >
          <code
            class="keyCode"
          >
            workbench.action.selectTheme
          </code>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
